### PR TITLE
Update the biome data to match 1.15

### PR DESCRIPTION
**Pull request by bemk**
I created a new error message about old biome data if the old format is found. This can be resolved by starting the minecraft server with the "--forceUpgrade" option.

### CODE REVIEW

File: src/mapcraftercore/mc/chunk.cpp [hunk 1]
```diff
@@ -178,15 +178,18 @@ bool Chunk::readNBT(mc::BlockStateRegistry& block_registry, const char* data, si
...
+	} else if (level.hasArray<nbt::TagIntArray>("Biomes", BIOMES_ARRAY_SIZE)) {
 		const nbt::TagIntArray& biomes_tag = level.findTag<nbt::TagIntArray>("Biomes");
 		std::copy(biomes_tag.payload.begin(), biomes_tag.payload.end(), biomes);
 	} else if (level.hasArray<nbt::TagByteArray>("Biomes", 0)
 			|| level.hasArray<nbt::TagLongArray>("Biomes", 0)) {
-		std::fill(biomes, biomes + 256, 0);
+		std::fill(biomes, biomes + BIOMES_ARRAY_SIZE, 0);
+	} else if (level.hasArray<nbt::TagByteArray>("Biomes", 256) || level.hasArray<nbt::TagIntArray>("Biomes", 256)) {
+		LOG(WARNING << "Out dated chunk " << chunkpos << ": Old biome data found!";
+	}
```
Comment (poebel) [inline src/mapcraftercore/mc/chunk.cpp]: Missing ")" after "LOG(WARNING"
And the "}" is one too much

Comment (bemk) [reply src/mapcraftercore/mc/chunk.cpp]: Dang, I thought I'd fixed that one before sending it out

**Comment by JulienTant** [comment]
Looks like this code now exists in the code base. Can we close this PR ?

**Comment by elamperti** [comment]
Yes @JulienTant, thank you! And thanks @bemk for your contribution as well :)

**Pull request closed without merging**